ginga/misc/plugins/MultiDim.py

```diff
@@ -410,7 +418,15 @@ def set_naxis(self, idx, n):
             self.naxispath[m] = idx
             self.logger.debug("m=%d naxispath=%s" % (m, str(self.naxispath)))
 
-            image.set_naxispath(self.naxispath)
+            if self.scaling_enabled:
+                curr_max = np.amax(image.get_mddata()[idx])
+                curr_min = np.amin(image.get_mddata()[idx])
+
```

Comment:
`amax` and `amin` will propagate NaNs.  I think it may be better to use `nanmax` and `nanmin`.